video/out/wayland_common.h

```diff
@@ -73,11 +73,25 @@ struct vo_wayland_state {
     struct xdg_wm_base      *wm_base;
     struct xdg_toplevel     *xdg_toplevel;
     struct xdg_surface      *xdg_surface;
+    struct wp_presentation  *presentation;
+    struct wp_presentation_feedback *feedback;
     struct zxdg_decoration_manager_v1 *xdg_decoration_manager;
     struct zxdg_toplevel_decoration_v1 *xdg_toplevel_decoration;
     struct zwp_idle_inhibit_manager_v1 *idle_inhibit_manager;
     struct zwp_idle_inhibitor_v1 *idle_inhibitor;
 
+    /* Presentation Feedback */
+    int64_t ust;
+    int64_t msc;
+    int64_t sbc;
```

Comment:
Is there any reason for using signed here over unsigned? Does `sbc` ever get decreased? Not that I'm particularly worried about a 64 bit overflow in a frame counter or anything.

Reply:
The only real reason is because `vo_vsync_info` signed integers (which is that way because `GLX_OML_sync_control` uses signed integers). I guess in OML those values can be negative, but in the wayland protocol they should only ever be positive as far as I understand it (and nah `sbc` never decreases).

Reply:
I don't think they can or should ever be negative. Signed type reduces the chance of accidentally converting other values to unsigned, though.